test/staticRender-spec.js

```diff
@@ -1,10 +1,11 @@
-import { describeWithDOM } from './_helpers';
+import { describeWithDOM, describeIf } from './_helpers';
 import React from 'react';
 import { expect } from 'chai';
 import { render } from '../src/';
+import { REACT013 } from '../src/version';
 
 describeWithDOM('render', () => {
-  describe('context', () => {
+  describeIf(!REACT013, 'context', () => {
```

Comment:
@hunterbmt it doesn't look like this `describeIf` block is wrapping all the new context tests